app/Http/Controllers/BuildController.php

```diff
@@ -381,6 +381,37 @@ public function apiBuildSummary(): JsonResponse
             $response['previousbuild'] = $previous_response;
         }
 
+        // Next build
+        if ($next_buildid > 0) {
+            $next_build = new Build();
+            $next_build->Id = $next_buildid;
+            $next_build->FillFromId($next_build->Id);
+
+            $next_response = [];
+            $next_response['buildid'] = $next_buildid;
+
+            // Update
+            $next_build_update = new BuildUpdate();
+            $next_build_update->BuildId = $next_build->Id;
+            $next_build_update->FillFromBuildId();
+            $next_response['nupdateerrors'] = $next_build_update->GetNumberOfErrors();
+            $next_response['nupdatewarnings'] = $next_build_update->GetNumberOfWarnings();
+
+            // Configure
+            $next_response['nconfigureerrors'] = $next_build->GetNumberOfConfigureErrors();
+            $next_response['nconfigurewarnings'] = $next_build->GetNumberOfConfigureWarnings();
+
+            // Build
+            $next_response['nerrors'] = $next_build->GetNumberOfErrors();
+            $next_response['nwarnings'] = $next_build->GetNumberOfWarnings();
+
+            // Test
+            $next_response['ntestfailed'] = $next_build->GetNumberOfFailedTests();
+            $next_response['ntestnotrun'] = $next_build->GetNumberOfNotRunTests();
+
+            $response['nextbuild'] = $next_response;
```

Comment:
This can be entirely inlined.  Doing so makes it more clear what fields are present in the return array.

Something along the lines of this:
```
response['nextbuild'] = [
    'buildid' => $next_buildid,
    'nupdateerrors' => $next_build_update->GetNumberOfErrors(),
    'nupdatewarnings' => $next_build_update->GetNumberOfWarnings(),
    ...
];
```

Reply:
That seems reasonable to me.